client/tests/cypress/integration/components/HomePage.spec.jsx

```diff
@@ -1,15 +0,0 @@
-describe('Home Page', () => {
```

Comment:
Why did you delete this?

Reply:
It has the same test as `login.spec.js` file.